src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -162,7 +183,13 @@ public RTCPTCCPacket(long senderSSRC, long sourceSSRC,
         int packetCount
             = 1 + RTPUtils.sequenceNumberDiff(last.getKey(), firstSeq);
 
+        // Temporary buffer to store the fixed fields (8 bytes) and the list of
+        // packet status chunks (see the format above). The buffer be longer
```

Comment:
nit: think there's a typo here: "The buffer be longer than needed" -> "The buffer may be longer than needed" (i think?)